third_party/maya/lib/usdMaya/JobArgs.h

```diff
@@ -168,12 +171,30 @@ struct JobExportArgs
     PXRUSDMAYA_API
     static const VtDictionary& GetDefaultDictionary();
 
+    PXRUSDMAYA_API
```

Comment:
For style consistency with the rest of the API, could these be "AddFilteredTypeName", "GetFilteredTypeIds", and "ClearFilteredTypeIds"?

Reply:
Also, could we add docs to AddFilteredTypeName about the inherited type behavior, e.g:
```
/// Adds type name to filter out during export. This will also add all 
/// inherited types (so if you exclude "constraint", it will also exclude 
/// "parentConstraint")
```